### DISCUSSION TITLE
connecting vsomeip to multiple NICs via the unicast parameter

### DISCUSSION THREAD

**Discussion by mogar** [COVESA/vsomeip#706] [Q&A]
I have a setup where we have multiple network interfaces that have SOME/IP devices on them. I'd like to run vsomeip and connect to all of those interfaces. Right now I can choose between them by changing the `unicast` parameter in my config file. Is it possible to connect to multiple unicast interfaces?

Alternatively, can I run multiple instances of vsomeip and connect each to a different unicast interface?

**Comment by NagarjunaReddyG5262** [comment]
Hi Did you get aby solution for this?
